src/Miningcore/DaemonInterface/DaemonClient.cs

```diff
@@ -79,7 +79,8 @@ public DaemonClient(JsonSerializerSettings serializerSettings, IMessageBus messa
 
         protected void PublishTelemetry(TelemetryCategory cat, TimeSpan elapsed, string info, bool? success = null, string error = null)
         {
-            messageBus.SendMessage(new TelemetryEvent(server, poolId, cat, info, elapsed, success));
+            //TODO: Commented out to see if it stops memory leak
```

Comment:
i don't think we need this change after all...

Reply:
Reverted it